### DISCUSSION TITLE
Error Exception page isn't working

### DISCUSSION THREAD

**Discussion by vishalmicrolevel** [laravel/framework#45241] [General]
Hello

The error exception page is not working.
when an error came the page isn't working  message came and did not display any errors.

I have  "laravel/framework": "^8.26.1",
please give me a solution.

**Comment by Cellard** [comment]
See https://laravel.com/docs/9.x/errors#custom-http-error-pages

**Comment by perisicnikola37** [comment]
Set APP_DEBUG to `true` in your `.env` file
